charts/studio/values.yaml

```
@@ -24,6 +24,14 @@ global:
   #
   customCaCert: ""
 
+  # -- Studio: Additional environment variables for all pods
+  envVars: []
+  # Example:
+  # envVars: [
+  #   - name: DEBUG
+  #     value: True
+  # ]
```

Let's make it a dict :-)

```suggestion
  envVars: {}
  # Example:
  # envVars: 
  #   - name: DEBUG
  #     value: True
```